src/NuGet.Core/NuGet.Commands/PackCommandRunner.cs

```diff
@@ -419,7 +613,7 @@ internal static string GetInputFile(PackArgs packArgs, IEnumerable<string> files
                     }
                     goto default;
                 default:
-                    throw new ArgumentException("Please specify a nuspec or project file to use");
+                    throw new ArgumentException("Please specify a nuspec, project.json, project file to use");
```

Comment:
Good catch - changing it.

---
In reply to: [58121029](https://github.com/NuGet/NuGet.Client/pull/432#discussion_r58121029) [](ancestors = 58121029)